docker-compose/_posts/2017-07-05-building-a-compose-file-for-runnable.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Building a Compose File for Runnable
+short: Building a Compose File for Runnable
+category: docker-compose
+order: 4
+---
+
+Runnable can deploy full-stack environments for branches of any repository that contains a Dockerfile. Let’s go over some practices you should follow in your Compose files to make this work for your team.
+
+---
+
+## Compose File Organization Patterns
+
+There are two patterns that teams tend to follow for organizing their Compose files:
+
+1. A centralized approach, with a single Compose file in a central repository that defines the whole stack.
+2. A decentralized approach, in which each service’s repository has its own Compose file that defines the environment it needs.
+
+Runnable works with either pattern because of how it makes use of the build feature in Compose.
+
+---
+
+## Using `build` to Create Environments for Your Branches
+
+Environments include a container for each service in your Compose file. Services can be based on images or built from a Dockerfile. A service that specifies a build enables us to launch environments for its branches.
```

Comment:
I don't think webhooks are relevant to what we're trying to communicate in this section, which is that we can launch environments for branches of any service that uses `build`. I took the first part of this suggestion and made an update to the last sentence in [the Google doc](https://docs.google.com/a/runnable.com/document/d/1K4lPBHUJdgy39xuqMFCM1i6Sn0VxZlKjilP7HPlU-rI/edit?usp=sharing).